scripts/kconfig/generate_header.py

```diff
@@ -10,6 +10,63 @@
 # SPDX-License-Identifier: ISC
 # Refer to scripts/kconfig/LICENSE.kconfiglib for the permission notice.
 import kconfiglib
+import re
+
+class Acrn_config(kconfiglib.Kconfig):
+    help_regex = re.compile("64-bit[\s\n]+integer")
+    def __init__(self, filename="Kconfig", warn=True, warn_to_stderr=True,
+          encoding="utf-8"):
+        kconfiglib.Kconfig.__init__(self, filename, warn, warn_to_stderr, encoding)
+
+    def write_autoconf(self, filename,
+          header="/* Generated by Kconfiglib (https://github.com/ulfalizer/Kconfiglib) */\n"):
+
+        guard_begin = "#ifndef __HV_KCONFIG__\n#define __HV_KCONFIG__\n"
+        guard_end = "#endif"
+
+        with open(filename, "w") as f:
+            f.write(header)
+            f.write(guard_begin)
+
+            for sym in self.defined_syms:
+                if sym.config_string in ("",None):
```

Comment:
This is awesome!

I had honestly never considered that `Symbol.config_string` can be used as a "hook" to `_write_config` in this way, and so not having access to `_write_to_conf` always felt like an icky API flaw. It just works out perfectly though.

The documentation now hints to it as well: https://github.com/ulfalizer/Kconfiglib/commit/bde4cc71fc01f209e1df0916843d351be201f78e

Thanks!